### PR TITLE
SC-383: Minimum update from 2.1.3 (R, Py, deploy)

### PR DESCRIPTION
**Pull request by xschildw**
This PR starts from 2.1.3 and just updates R and Python. It also moves from travis to GHA for deployment.

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        with:
+          version: latest
+      - name: packer validate
+        run: |
+          pushd src
+          packer validate -var ImageName=validate template.json
+  deploy:
+    name: Deploy to AWS org-sagebase-imagecentral
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/2.1.3.1' || startsWith(github.ref, 'refs/tags/v') }}
```
Comment (xschildw) [inline .github/workflows/deploy.yaml]: Changed from master, I think the tag prefix should be 'refs/tags/v2.1.3'.

Comment (zaro0508) [reply .github/workflows/deploy.yaml]: you should  create a `v2.1.3` branch and set this to `ref/heads/v2.1.3`.  Then you should target this PR against the `v2.1.3` branch

File: src/playbook.yaml [hunk 2]
```diff
@@ -47,6 +47,13 @@
           echo 'LoadModule python_module /usr/lib/apache2/modules/mod_python.so' >> /etc/apache2/mods-available/python.load
         chdir: /tmp/mod_python_source
 
+    - name: Install python3.8 (highest we can get on U18.04)
+      cmd: |
+        add-apt-repository -y ppa:deadsnakes/ppa
+        apt update
+        apt install -y python3.8
+        update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.6 1
+        update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 2
```
Comment (zaro0508) [inline src/playbook.yaml:56]: does this make python3.8 the default python?  don't we also need to set the default version with `update-alternatives --set python3 /usr/bin/python3.8`?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (xschildw) [reply .github/workflows/deploy.yaml]: OK

File: src/playbook.yaml (hunk 2 above)
Comment (xschildw) [reply src/playbook.yaml:56]: The higher priority makes it the default but no harm in being explicit, will do.

File: .github/workflows/deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        with:
+          version: latest
+      - name: packer validate
+        run: |
+          pushd src
+          packer validate -var ImageName=validate template.json
+  deploy:
+    name: Deploy to AWS org-sagebase-imagecentral
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/2.1.3' || startsWith(github.ref, 'refs/tags/v2.1.3') }}
```
Comment (xschildw) [inline .github/workflows/deploy.yaml]: Used 2.1.3 as GH warns about potential confusion if using same branch and tag name...

Comment (zaro0508) [reply .github/workflows/deploy.yaml]: branch should be `v2.1.3` and condition should be `github.ref == 'refs/heads/v2.1.3'`

also i think you can leave tags condition as `startsWith(github.ref, 'refs/tags/v')`

**Review by zaro0508 — CHANGES_REQUESTED** [review]

**Review by zaro0508 — APPROVED** [review]

**Pull request merged**